Cotecna.OpenSource.Domain.Core/Common/Contracts.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Cotecna.OpenSource.Domain.Core
+{
+    public static class Contracts
```

Comment:
could we add header documentation in public exposed contracts and methods?

Reply:
Added comments to public metods and classes